docs(changelog): organize headings and update style for improved look

monicawheeler opened the pull request:
# Description
* Updated CSS for style changes and modified HTML to fit accordingly. 
* Modified the JS to include descriptions for sections that are present and along with those sections.

## Screenshots
| Before    | After |
| -------- | ------- |
|  ![Screenshot 2024-05-07 at 10 25 59 AM](https://github.com/Kajabi/pine-icons/assets/24237393/14d16c82-66b9-4747-a955-f024f57bfcf9) |  ![Screenshot 2024-05-09 at 3 04 43 PM](https://github.com/Kajabi/pine-icons/assets/24237393/aa7ac45c-74a6-4c93-9559-f4afe8051f35) |


## Type of change

Please delete options that are not relevant.
If your type of change is not present, add that option.

- [x] Documentation improvement
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests you've added and run to verify your changes.
Provide instructions so that we can reproduce.
Please also list any relevant details for your test configuration.

- [ ] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [ ] tested manually
- [ ] other:

**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

# Checklist:

If not applicable, leave options unchecked.

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

anechol reviewed (approved):
This is definitely way easier to read. I don't think this needs to look extremely elaborate in terms of styling. Just adding the bolded text in the info section and the color to the version number makes the important information stand out as needed.

pixelflips commented:
This looks great. My only suggestion would be maybe to adjust the ordering of the info section. Since it's static and takes up a lot of space at the top of the page, could we consider moving it below the tables or possibly having a two-column layout? Tables on the left and info on the right. 

Just some thoughts, but I'll let you decide since it's already a nice improvement. I'd vote for just straight system fonts (no google), but that's just my preference. ;)

monicawheeler commented:
@pixelflips - I removed the google fonts and modified the code to show the descriptions with each table, while keeping a single col layout. I updated the screenshot. Let me know what you think! Thanks for the feedback!

pixelflips commented on scripts/figma_icons/changelog-template.html:
```diff
@@ -2,11 +2,16 @@
+      max-width: 1000px;
+      margin-right: auto;
+      margin-left: auto;
       font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```
❤️

pixelflips reviewed (approved):
Nice improvement! LGTM! :shipit:

ju-Skinner commented on scripts/figma_icons/figma-icon-export.ts:
```diff
@@ -647,31 +650,32 @@ const processData = async (rootDir: string, config: FigmaIconConfig) => {
   if (n.length > 0) {
     created = await Promise.all(n.map((path) => { return buildBeforeAndAfterSvg('', path)}));
-    created = buildHTMLSection('Added', created);
+    created = buildHTMLSection('Added', created, 'New icons introduced in this version. You will not see them in the "before" column because they didn not exist in the previous version.');
```
Have a typo here. Remove the `n` in the word `didn`.
